### PR TITLE
[ACR-1811] Deprecate underutilized fields

### PR DESCRIPTION
**Pull request by MinjeAhn**
### API Changes ###

ACR: [ACR-1811](https://jira.sec.samsung.net/browse/ACR-1811)

### CODE REVIEW

**Comment by journey2w** [comment]
Please checkout the webpage https://docs1.stg.tizen.org/staging/1919

**Comment by journey2w** [comment]
Please checkout the webpage https://docs1.stg.tizen.org/staging/1919

File: docs/application/native/guides/multimedia/media-content.md [hunk 1]
```diff
@@ -1058,35 +1058,32 @@ To access media item information:
...
-                   video_meta_get_artist(video_handle, &artist);
-                   video_meta_get_album(video_handle, &album);
-                   video_meta_get_album_artist(video_handle, &album_artist);
-                   video_meta_get_duration(video_handle, &duration);
+                   audio_meta_get_artist(audio_handle, &artist);
+                   audio_meta_get_album(audio_handle, &album);
+                   audio_meta_get_album_artist(audio_handle, &album_artist);
 
-                   dlog_print(DLOG_DEBUG, LOG_TAG, "This is a video");
+                   dlog_print(DLOG_DEBUG, LOG_TAG, "This is a music");
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/media-content.md:1076]:
```suggestion
                   dlog_print(DLOG_DEBUG, LOG_TAG, "This is an audio file");
```

**Review by safir-srbd — COMMENTED** [review]
@MinjeAhn Hello. Hope you are doing well!

I have gone through the changes made, and have a small suggestion. Please check and implement it.

**Comment by journey2w** [comment]
Please checkout the webpage https://docs1.stg.tizen.org/staging/1919

**Review by safir-srbd — APPROVED** [review]
@MinjeAhn Thank you for making the suggested change. I am approving this PR for merging.

**Pull request merged**